projects/can_transceiver/inc/can_transceiver.h

```diff
@@ -99,12 +100,3 @@ class CanbusIntf : public CanTransceiver
      */
     void send(const CanFrame & frame) const;
 };
-
```

Comment:
We'll need to keep this for future implementation. You can remove the `public rclcpp::Node`.

Reply:
Added back:
```
/**
 * Implementation of CAN Transceiver that interfaces with the simulator
 *
 */
class CanSimTransceiver  : public CanTransceiver
{
    void receive();
};

```